### ISSUE TITLE
issues running

### ISSUE THREAD

**Issue by bogorman** [sc1f/perspective-order-book#1]
Hello. 
I am getting this in the browser console when I try to run. 

index.js:57 Uncaught (in promise) 'Table' object has no attribute 'get_limit'

 Also could you include the layout.json?

**Comment by bogorman** [comment]
Installing older version of perspective-python gets rid of the error. I still just see a blank screen because the layout is blank I am guessing.

**Comment by sc1f** [comment]
Hi - this example hasn't been updated in some time so it may be a little stale with regards to the Perspective API. I'll go ahead and update it to use 1.0.1 (the latest perspective-python release). The error message you're seeing seems like it's being thrown in Javascript, so maybe the perspective version installed from NPM is off? The easiest way to get it working is to make sure that both versions of Perspective on the client and the server are the same, so both are at 1.0.1.

In terms of the layout, it seems like I forgot to include a default layout; you should be able to run this snippet in the Chrome console to restore a default:

```javascript
window.workspace.restore({
    detail: {
        main: {
            currentIndex: 0,
            type: "tab-area",
            widgets: ["flat"],
        },
    },
    viewers: {
        flat: {table: "order_book"},
    },
})
```

**Comment by bogorman** [comment]
yes downgrading the server side to 0.10.2 worked for me. I'll try the layout. thanks
